locales/es.json

```diff
@@ -988,8 +988,7 @@
 			},
 			"tasks": {
 				"button": {
-					"createTask": "Crear una tarea",
-					"privateTasks": "Tareas privadas"
```

Comment:
Why is the key deleted? (privateTasks)

Reply:
the key is not in used. so I cleaned it up.